### PR TITLE
[Fix] fixing primitive type issue

### PR DESCRIPTION
**Pull request by grahamjenson**
Small issue with the Type v.s. Primitive type.

### CODE REVIEW

**Comment by sam-goodwin** [comment]
Fixed in https://github.com/awslabs/goformation/pull/166

**Comment by grahamjenson** [comment]
@sam-goodwin #166 did not fix the Authorizers issue. I will update this PR to fix.

File: generate/sam-2016-10-31.json [hunk 1]
```diff
@@ -752,7 +752,7 @@
                 "Authorizers": {
                     "Documentation": "https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api-auth-object",
                     "Required": false,
-                    "Type": "String",
+                    "PrimitiveType": "Json",
```
Comment (sam-goodwin) [inline generate/sam-2016-10-31.json]: I think the type is actually more defined than `Json`: 

https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api-auth-object

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-authorizer.html

Comment (grahamjenson) [reply generate/sam-2016-10-31.json]: Yeah, Json  in this location is a placeholder for a more complicated object (beyond my knowledge) to be constructed.

**Pull request merged**